### PR TITLE
Add script to compare vcf with positive controls

### PR DESCRIPTION
**Pull request by adrosenbaum**
added:

	- lib/MIB/Program/Varg.pm

### This PR fixes:

- 

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [x] Code review
- [x] New code is executed and covered by tests
- [x] Tests pass

### CODE REVIEW

**Comment by adrosenbaum** [comment]
Hi, I've created a new repo under my user https://github.com/adrosenbaum/varg containing a tool for comparing a vcf against a truth-set. It is installable via PyPI (pip install varg).

File: t/varg_compare.t [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    infile_path_truth => {
+        input           => q{test_infile_truth.vcf},
+        expected_output => q{--truth-set} . $SPACE . q{test_infile_truth.vcf}
+    },
+    infile_path_vcf => {
+        input           => q{test_infile_vcf.vcf},
+        expected_output => q{--variants} . $SPACE . q{test_infile_vcf.vcf}
+    }
+);
+
```
Comment (henrikstranneheim) [inline t/varg_compare.t]: Duplicate the %required argument with a %specific_argument. That enables individual tests of each argument while making sure that the required arguments always are present.

**Comment by henrikstranneheim** [comment]
@adrosenbaum  Travis fails on your new module: Run your test command locally using perl t/varg_compare.t. Does it pass?

**Comment by adrosenbaum** [comment]
@henrikstranneheim found a syntax error, might be that 😄

**Comment by henrikstranneheim** [comment]
Great! Push again and let's see what travis says

**Pull request closed without merging**